shared/reducers/config.js

```diff
@@ -8,6 +8,7 @@ import type {Config, GetCurrentStatusRes, ExtendedStatus} from '../constants/typ
 
 export type ConfigState = {
   globalError: ?Error,
+  daemonError: ?string,
```

Comment:
I think it's better to pass Error objects around. In this case, it's not a big difference but a good convention to follow.

On a side note, in the past we had tons of problems with converting Error's to strings (mostly because the msgpack rpc library uses a `desc` property for it's errors instead of the standardized `message`), so unifying around a single Error object (that other 3rd libraries also use) seems to work well.